spec/components/contents/resource_component_spec.rb

```diff
@@ -3,13 +3,56 @@
 require 'rails_helper'
 
 RSpec.describe Contents::ResourceComponent, type: :component do
-  pending "add some examples to (or delete) #{__FILE__}"
-
-  # it "renders something useful" do
-  #   expect(
-  #     render_inline(described_class.new(attr: "value")) { "Hello, components!" }.css("p").to_html
-  #   ).to include(
-  #     "Hello, components!"
-  #   )
-  # end
+  let(:component) { described_class.new(resource: resource, resource_counter: 1, object_id: 'druid:kb487gt5106', viewable: true) }
+  let(:rendered) { render_inline(component) }
+  let(:resource) do
+    instance_double(Cocina::Models::FileSet,
+                    type: type,
+                    externalIdentifier: 'https://cocina.sul.stanford.edu/fileSet/bb573tm8486-bc91c072-3b0f-4338-a9b2-0f85e1b98e00',
+                    version: 3,
+                    label: 'Object 1',
+                    structural: instance_double(Cocina::Models::FileSetStructural, contains: [file]))
+  end
+  let(:file) do
+    Cocina::Models::File.new(type: Cocina::Models::ObjectType.file,
+                             filename: 'example.tif',
+                             label: 'example.tif',
+                             externalIdentifier: 'https://cocina.sul.stanford.edu/file/b7cdfa7a-6e1f-484b-bbb0-f9a46c40dbb4',
+                             hasMimeType: 'image/tiff',
+                             version: 3,
+                             access: {
+                               view: 'world',
+                               download: 'world'
+                             },
+                             administrative: {
+                               publish: true,
+                               sdrPreserve: true,
+                               shelve: true
+                             },
+                             presentation: presentation)
+  end
+  let(:type) { 'https://cocina.sul.stanford.edu/models/resources/image' }
+  let(:presentation) { { height: 11_839, width: 19_380 } }
+
+  it 'renders the component' do
```

Comment:
Can you wrap this in "with an image" context?

Reply:
Added that (and did similar on `file_component_spec.rb`)